### PR TITLE
Add typescript definitions

### PR DESCRIPTION
**Pull request by mc-petry**
Fix https://github.com/agorf/react-use-sync-scroll/issues/4

### CODE REVIEW

**Comment by agorf** [comment]
Hi @mc-petry Thanks for contributing.

Since I don't know or use TypeScript myself, could you explain what this does for TypeScript users and how it benefits the project?

Thanks

**Comment by mc-petry** [comment]
This type definitions adds intelliSense and static type checking.

image

more info you can find at https://github.com/microsoft/TypeScript

File: package.json [hunk 1]
```diff
@@ -35,12 +35,14 @@
...
+    "rollup-plugin-copy": "^3.1.0"
   },
   "peerDependencies": {
     "react": ">=16.8"
   },
   "files": [
     "dist"
-  ]
+  ],
+  "typings": "dist/index.d.ts"
```
Comment (agorf) [inline package.json]: Please use `types` here

File: yarn.lock [hunk 2]
```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```
Comment (agorf) [inline yarn.lock]: All integrity lines seem to go away. Any reason for this?

File: package.json (hunk 1 above)
Comment (mc-petry) [reply package.json]: Renamed

File: yarn.lock (hunk 2 above)
Comment (mc-petry) [reply yarn.lock]: Sorry. May be i have old version of yarn (i use npm). Yarn did it himself. I'm revert yarn.lock to previous version

Comment (agorf) [reply yarn.lock]: np!

**Comment by agorf** [comment]
Merged with https://github.com/agorf/react-use-sync-scroll/commit/1424a3f2101e3457c47be660ba8acd52642aea25

**Comment by agorf** [comment]
Thanks!

**Pull request closed without merging**